cmd/kubectl-access/kubectl-access.go

```diff
@@ -223,9 +223,12 @@ func (ac *accessCommand) Request(cmd *cobra.Command, args []string) error {
 	}
 
 	currentContext := rawConfig.CurrentContext
-	if ac.genericOptions.Context != nil {
+	if ac.genericOptions.Context != nil && *ac.genericOptions.Context != "" {
```

Comment:
Is there a reason for doing one check by calling the `ac` and then doing the other with a pointer? I haven't used cobra for some time and might be the way to check this things out, but it looks a bit odd.

Reply:
Ok ok. I gave a second check, and just noticed that the `genericOptions` is a pointer, therefore it totally makes sense to do that. So this is fine in my opinion